src/openms/include/OpenMS/ANALYSIS/ID/FragmentIndex.h

```diff
@@ -39,6 +39,41 @@ namespace OpenMS
       float precursor_mz;  // charge 1 precursor m/z
     };
 
+    struct sPeptide
```

Comment:
I would suggest to just call it Peptide for consistency. It is in the FragmentIndex class scope so there is no danger to confuse it with some other Peptide datastructure that might be around.